src/main/java/com/mparticle/kits/ApptentiveKit.java

```diff
@@ -170,20 +184,18 @@ public void removeUserIdentity(MParticle.IdentityType identityType) {
 
 	@Override
 	public List<ReportingMessage> logEvent(MPEvent event) {
-		Map<String, String> customData = event.getInfo();
-		if (customData != null) {
-			Apptentive.engage(getContext(), event.getEventName(), Collections.<String, Object>unmodifiableMap(customData));
-		} else {
-			Apptentive.engage(getContext(), event.getEventName());
-		}
+		engage(getContext(), event.getEventName(), event.getInfo());
```

Comment:
nitpick: `MPEvent#getInfo()` is deprercated, use `getCustomAttributes()` instead

Reply:
Yes, @willpassidomo, we intentionally set the flag to true as we believe that will be the best experience for customers.  In future, we may ask you to allow customers to turn it off on their connection.  We'll track the deprecated api usage for a future release.

Reply:
sounds good! Good to merge when you're ready :)